Lykke.Service.LykkeService/tests/Lykke.LykkeType.LykkeService.Tests/ClientHttpPathsAreMatchedInControllersTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Lykke.Service.LykkeService;
+using Lykke.Service.LykkeService.Client;
+using Microsoft.AspNetCore.Mvc;
+using Refit;
+using Xunit;
+
+namespace Lykke.LykkeType.LykkeService.Tests
+{
+    public class ClientHttpPathsAreMatchedInControllersTest
+    {
+        private readonly Type _routeAttrType = typeof(RouteAttribute);
+        private readonly List<Type> _refitAttrs = new List<Type>
+        {
+            typeof(GetAttribute), typeof(PostAttribute), typeof(PutAttribute), typeof(DeleteAttribute)
+        };
+        private readonly List<Type> _httpAttrs = new List<Type>
+        {
+            typeof(HttpGetAttribute), typeof(HttpPostAttribute), typeof(HttpPutAttribute), typeof(HttpDeleteAttribute)
+        };
+
+        [Fact]
+        public void CheckRoutesInControllersTest()
+        {
+            var clientInterface = typeof(ILykkeServiceClient);
+
+            var apiInterfaces = clientInterface
+                .GetProperties()
+                .Where(p => p.CanRead && p.PropertyType.IsInterface)
+                .Select(p => p.PropertyType)
+                .ToList();
+            var controllers = Assembly.GetAssembly(typeof(Startup))
```

Comment:
Some controllers may be declared in external nuget packages like `IsAlive` controller.

Reply:
This is for business controllers, that inherit client api interfaces. Service controllers, like IsAlive, usually has well-known interface and don't provide client functionality. We can change it later if refuired.